coalib/coala_delete_orig.py

```diff
@@ -1,21 +1,17 @@
+import logging
 import os
 
-from pyprint.ConsolePrinter import ConsolePrinter
-
 from coalib.output.Logging import configure_logging
-from coalib.output.printers.LogPrinter import LogPrinter
 from coalib.parsing import Globbing
+from coalib.parsing.Globbing import glob_escape
 from coalib.settings.ConfigurationGathering import get_config_directory
 from coalib.settings.Section import Section
-from coalib.parsing.Globbing import glob_escape
 
 
-def main(log_printer=None, section: Section=None):
+def main(section: Section=None):
```

Comment:
Are you sure doing this doesn't cause any deprecation problems. No external parties use it.